arkane/ess/psi4.py

```diff
@@ -0,0 +1,332 @@
+        rot = []
+        inertia = []
+        unscaled_frequencies = []
+        e0 = 0.0
+
+        if optical_isomers is None or symmetry is None:
+            _optical_isomers, _symmetry, _ = self.get_symmetry_properties()
+            if optical_isomers is None:
+                optical_isomers = _optical_isomers
+            if symmetry is None:
+                symmetry = _symmetry
+
+        with open(self.path, 'r') as f:
+            line = f.readline()
+            while line != '':
+                # Read spin multiplicity if not explicitly given
+                if 'Charge       =' in line:
+                    charge = int(float(line.split()[2]))
+                    logging.debug(
+                        'Conformer {0} is assigned a charge of {1}'.format(label, charge))
+                if 'Multiplicity =' in line and spin_multiplicity == 0:
+                    spin_multiplicity = int(float(line.split()[2]))
+                    logging.debug(
+                        'Conformer {0} is assigned a spin multiplicity of {1}'.format(label, spin_multiplicity))
+                # The rest of the data we want is in the Thermochemistry section of the output
+                if 'Harmonic Vibrational Analysis' in line:
+                    modes = []
+                    frequencies = []
+                    self.neg_frequencies = None
+                    while 'Thermochemistry Components' not in line:
+                        # This marks the end of the thermochemistry section
+                        if 'Thermochemistry Components' in line:
+                            break
+
+                        # Read vibrational modes
+                        if 'Freq [cm^-1]' in line:
+
+                            if len(line.split()) == 5:
+                                frequencies.extend([float(convert_i_to_neg(d)) for d in line.split()[-3:]])
+                            elif len(line.split()) == 4:
+                                frequencies.extend([float(convert_i_to_neg(d)) for d in line.split()[-2:]])
+                            elif len(line.split()) == 3:
+                                frequencies.extend([float(convert_i_to_neg(d)) for d in line.split()[-1:]])
+                        line = f.readline()
+                    # If there is an imaginary frequency, remove it
+                    if any([f < 0.0 for f in frequencies]):
+                        neg_frequencies = [f for f in frequencies if f < 0.0]
+                        frequencies = [f for f in frequencies if f > 0.0]
+                        self.neg_frequencies = neg_frequencies
+
+                    unscaled_frequencies = frequencies
+                    vibration = HarmonicOscillator(frequencies=(frequencies, "cm^-1"))
+                    # modes.append(vibration)
+                    freq.append(vibration)
+
+                    # Read the next line in the file
+                line = f.readline()
+
+                # Read the next line in the file
```

Comment:
Is this comment supposed to be here? What about the `line = f.readline()` on the next line?

Reply:
Those are carried over from qchem. I will clean up.